CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format is based on [Common Changelog](https://common-changelog.org).\
+Unlike Common Changelog, the `unreleased` section of [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) is preserved with the addition of a tag to specify which type of release should be published and to foster discussions about it inside pull requests. This tag should be one of the following: `[patch]`, `[minor]`, `[major]`. Ex `Unreleased [minor]`.
+
+## [Unreleased][minor]
```

Comment:
? 🤔 
```suggestion
## Unreleased [minor]
```

This seems to be a good demonstration we need to check in CI that the format is correct 😄

Reply:
Both formats are valid and supported by keep-a-changelog

Reply:
Yes, but we don't parse that format with our `sed` 😉